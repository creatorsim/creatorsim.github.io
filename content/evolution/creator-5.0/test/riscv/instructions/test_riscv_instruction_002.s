
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    li x5, 10
    addi x9, x5, -30       # 10 - 30
    addi x10, x9, 411      # x9 + 411
    addi x11, x10, 12      # x10 - 12
    #addi x11, x10, 42568  # x10 - 42568