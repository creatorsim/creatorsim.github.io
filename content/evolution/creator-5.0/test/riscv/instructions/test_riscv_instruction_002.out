
cr[PC]:0x10; ir[x5,t0]:0xa; ir[x9,s1]:0xffffffec; ir[x10,a0]:0x187; ir[x11,a1]:0x193; keyboard[0x0]:''; display[0x0]:''; 

